Assets/Scenes/Scripts/PlayerControl.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+[System.Serializable]
+public class Boundary{
+	public float xMin, xMax, zMin, zMax;
+}
+public class PlayerControl : MonoBehaviour {
+	public float speed;
+	public Boundary boundary;
+	public float tilt;
+	public GameObject shot;
+	public Transform shotspot;
+	public float fireRate;
+	private float nextfire;
+
+	public GameObject player_ex;
+
+	public static int health;
+	public GameObject PlayerHealthBar;
+	public Sprite sp_08;
+	public Sprite sp_06;
+	public Sprite sp_04;
+	public Sprite sp_02;
+	public Sprite sp_00;
+
+	public static bool damaged;
+
+	// Use this for initialization
+	void Start () {
+		damaged = false;
+		health = 5;
+	}
+	public void Damage(){
+		health--;
+
+		Sprite sp = sp_00;
+		switch (health){
+		case 4:
+			sp = sp_08;
+			break;
+		case 3:
+			sp = sp_06;
+			break;
+		case 2:
+			sp = sp_04;
+			break;
+		case 1:
+			sp = sp_02;
+			break;
+		case 0:
+			GameController.GameOver ();
+			Instantiate (player_ex, this.transform.position, this.transform.rotation);
+			Destroy (PlayerHealthBar);
+			Destroy (this.gameObject);
+			return;
+		}
+		PlayerHealthBar.GetComponent<SpriteRenderer> ().sprite = sp;
+	}
+	void FixedUpdate(){
+		if (!GameController.gameover) {
```

Comment:
I know it seems weird but I would prefer if you did this
```suggestion
		if (GameController.gameover == false) {
```
This makes it a little more readable at first glance even if it seems counterintuitive